portal/README.md

```diff
@@ -0,0 +1,45 @@
+<img src="https://github.com/att/qujata/assets/7979328/199a9c84-840b-415e-a221-621c22184ad2" width="300">
+
+## Description
+This is a `Post Quantum Cryptography` tool,
+<br/>
+using [React](https://github.com/facebook/create-react-app) framework TypeScript starter repository.
+
+
+## Pre-requisite
+[Node.js](#https://nodejs.org/en/download) (includes npm) has to be installed on your system in order to run this project.
+
+## Installation
+
+1. To start, clone the qujata repository.
+```bash
+git clone https://github.com/att/qujata.git
+cd qujata/portal
+```
+2. Install yarn:
```

Comment:
we used '.' to end the line on 1 and ':' on 2
let's keep a same convention for both